### PR TITLE
Add concept for priority and queueing strategy

### PR DESCRIPTION
**Pull request by alculquicondor**
#### What type of PR is this?

/kind documentation

#### Which issue(s) this PR fixes:

Part of #64

### CODE REVIEW

File: docs/concepts/cluster_queue.md [hunk 1]
```diff
@@ -53,6 +53,25 @@ namespaceSelector:
...
+You can set different queueing strategies in a ClusterQueue using the
+`.spec.queueingStrategy` field. The queueing strategy determines how workloads
+are ordered in the ClusterQueue and how they are re-queued after an unsuccessful
+[admission](.#admission) attempt.
+
+The following are the supported queueing strategies:
+
+- `StrictFIFO`: Workloads are ordered first by [priority](queued_workload.md#priority)
+  and then by `.metadata.creationTimestamp`. Older workloads that can't be
+  admitted will block admitting newer workloads even if they fit in the
```
Comment (cortespao) [inline docs/concepts/cluster_queue.md:67]:
```suggestion
  admitted will block newer workloads, even if the newer workloads fit in the
```

File: docs/concepts/cluster_queue.md [hunk 2]
```diff
@@ -53,6 +53,25 @@ namespaceSelector:
...
+  available quota.
+- `BestEffortFIFO`: Workloads are ordered the same way as `StrictFIFO`. However,
+  older workloads that can't be admitted will not block admitting newer
```
Comment (cortespao) [inline docs/concepts/cluster_queue.md:70]:
```suggestion
  older workloads that can't be admitted will not block newer
```

**Review by cortespao — APPROVED** [review]

File: docs/concepts/cluster_queue.md (hunk 2 above)
Comment (alculquicondor) [reply docs/concepts/cluster_queue.md]: Done

**Comment by alculquicondor** [comment]
squashed

/hold cancel

**Pull request merged**